### PR TITLE
CDDL files

### PR DESCRIPTION
**Pull request by GeorgeFlerovsky**

### CODE REVIEW

**Comment by nicolasayotte** [comment]
We agreed to add a bit of precision on the asset label types. Something like
```
bytes_32 = bytes .size (0..32)
label_utf8_32 = text .size (0..32) ; utf-8 encoded
label_hex_64 = text .size (0..64) ; hex encoded bytes_32
```
then merge.

**Comment by nicolasayotte** [comment]
This will allow the CSV format to support both types of labels, probably used a `version` tag to tell the user what to expect

**Comment by GeorgeFlerovsky** [comment]
> This will allow the CSV format to support both types of labels, probably used a `version` tag to tell the user what to expect

I split the CDDL file into two files (`version_1.cddl` and `version_2.cddl`). In version two, the `action` object sets `version` to `2`.

File: CIP-????/cddl/version_1.cddl [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+address = bytes
+
+; =========================================================================
+; Convenient type aliases
+
+; Fixed-size text and bytes
+label_32 = bytes_32 / text_32
+bytes_32 = bytes .size (0..32)
+text_32 = bytes .size (0..32)
```
Comment (nicolasayotte) [inline CIP-????/cddl/version_1.cddl]: text .size (0..32) ?

File: CIP-????/cddl/version_2.cddl [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+address = bytes
+
+; =========================================================================
+; Convenient type aliases
+
+; Fixed-size text and bytes
+label_32 = bytes_32 / text_32
+bytes_32 = bytes .size (0..32)
+text_32 = bytes .size (0..32)
```
Comment (nicolasayotte) [inline CIP-????/cddl/version_2.cddl]: text again

**Review by nicolasayotte — APPROVED** [review]

**Pull request merged**